test/react-native/features/fixtures/app/ios/exportOptions.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>teamID</key>
+    <string>7W9PZ27Y5F</string>
```

Comment:
I take it this is safe to leave here?

Reply:
I assumed so, since this was lifted from the test fixtures in the notifier repo